frontend/tests/e2e_tests/integration/99-settings.spec.ts

```diff
@@ -223,6 +223,8 @@ test.describe('Settings', () => {
       await isLoggedIn(page);
       await page.goto(`${baseUrl}ui/settings/my-account`);
```

Comment:
Not a breaking change (yet), but I think this URL was renamed from `my-account` to `my-profile`

Reply:
good catch, that seems to have never been aligned 🤨